hack/components/git-utils.sh

```diff
@@ -26,3 +26,7 @@ function git-utils::get_component_tag() {
         git describe --tags
     )
 }
+
+function git-utils::create_temp_path() {
+	mktemp -d -p .
```

Comment:
i think its better to create under /tmp
even that its deleted at the end, 
its cleaner and safer in case script crashes

Reply:
it's possible and I did that at first but eventually preferred this way. here's my reasoning:

if I create the temp folder inside `./tmp` then the TEMP_PATH  wil be (for example) `./tmp/tmp.JzzNzXFcEh/`
and the the trap `trap "rm -rf ${TEMP_DIR}" EXIT` will not delete the ./tmp folder. 
Of course I can do `trap "rm -rf ./tmp" EXIT`, but I prefer to only delete the folder i created, and not the entire ./tmp folder that may contain other stuff that are running now.

Again - only my personal preferece.

Reply:
totally meant a folder at /tmp
i.e /tmp/XXYY

Reply:
understood what you mean. will change.